<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ehcap_window</class>
 <widget class="QWidget" name="ehcap_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Echap</string>
  </property>
  <widget class="QPushButton" name="resume_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settings_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>Param√®tres</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
